blaze-server/src/test/scala/org/http4s/server/blaze/BlazeServerSpec.scala

```diff
@@ -17,6 +21,20 @@ class BlazeServerSpec extends ServerAddressSpec {
 
       true must_== true
     }
+
+    "Startup with a custom ExecutionContext" in {
+      val s = BlazeBuilder
+        .bindAny()
+        .withExecutionContext(
+          ExecutionContext.
+            fromExecutorService(
+              Executors.newFixedThreadPool(5)))
```

Comment:
Agreed. Couldn't find a way to confirm execution on provided executor in the tests. I'm going to remove this in a separate PR.